.eslintrc.json

```diff
@@ -1,3 +1,7 @@
 {
-    "extends": "airbnb-base"
-}
\ No newline at end of file
+    "extends": "airbnb-base",
+    "env": {
+        "browser": true,
+        "node": true
```

Comment:
Might want to consider having different ESLint configurations for the back-end and front-end sources so it detects incorrect Node usage in front-end code and vice versa.

Reply:
Probably the best option, I was just unsure on how to achieve that. I'll have a look